commands/user_e2e_test.go

```diff
@@ -136,3 +136,36 @@ func (s *MmctlE2ETestSuite) TestSearchUserCmd() {
 		s.Equal("Unable to find user '"+emailArg+"'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestUpdateUserEmailCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForSystemAdminAndLocal("admin and local can change basic user email", func(c client.Client) {
+		newEmail := "basicuser2@fakedomain.com"
+		err := updateUserEmailCmdF(c, &cobra.Command{}, []string{s.th.BasicUser2.Email, newEmail})
+		s.th.BasicUser2.Email = newEmail
+		s.Require().Nil(err)
+	})
+
+	s.Run("normal client doesn't have permission to change email of another client", func() {
+		s.th.LoginBasic2WithClient(s.th.Client)
+		newEmail := "basicuser-change@fakedomain.com"
+		err := updateUserEmailCmdF(s.th.Client, &cobra.Command{}, []string{s.th.BasicUser.Id, newEmail})
+		s.Require().EqualError(err, ": You do not have the appropriate permissions., ")
+		s.th.BasicUser.Email = newEmail
+	})
+
+	s.RunForAllClients("wrong number of arguments", func(c client.Client) {
```

Comment:
As this test case is not using the API directly, it can be part of the unit tests

Reply:
I saw the unit tests and those cases are contemplated. I'll remove this testcase